pandas/tests/groupby/test_function.py

```diff
@@ -1594,3 +1594,19 @@ def test_multiindex_group_all_columns_when_empty(groupby_func):
     result = method(*args).index
     expected = df.index
     tm.assert_index_equal(result, expected)
+
+
+def test_deprecated_keywords():
+    msg = (
+        "In the future version of pandas the arguments args"
+        "and kwargs will be deprecated for these functions"
+    )
+    with tm.assert_produces_warning(FutureWarning, match=msg):
+        df = DataFrame({"a": [1, 1, 2], "b": [3, 4, 5]})
+        gb = df.groupby("a")
+        assert gb.cummax(kwargs=1)
```

Comment:
I think using `kwargs` as the sample  keyword argument is a bit misleading. Something like `gb.cummax(incorrect_argument=`)` would probably be clearer.